README.md

```diff
@@ -182,7 +150,7 @@ null;
 
 undefined;
 ```
-  
+
 ## Return Values
```

Comment:
this section on return values would probably make more sense when learning about functions. ie "Functions have return values the same way that methods do in Ruby. However, unlike in Ruby, you must explicity declare..." 

Should we remove this section and move to a later README?